testng.xml

```diff
@@ -1,10 +1,13 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 <suite name="mobile-automation-framework">
-	<parameter name="os" value="${os}" />
-
+	<parameter name="os" value="android" />
+	<listeners>
+	<listener class-name="UITestFramework.retryLogic.TestListener"/> 
+	</listeners>
 	<test name="automation-tests">
+	<parameter name="os" value="android" />
```

Comment:
I am do not think, this line is required

Reply:
Removed